helm/idol-answerserver/templates/configmap-postgres.yaml

```
@@ -0,0 +1,20 @@
+# BEGIN COPYRIGHT NOTICE
+# Copyright 2023 Open Text.
+# 
+# The only warranties for products and services of Open Text and its affiliates and licensors
+# ("Open Text") are as may be set forth in the express warranty statements accompanying such
+# products and services. Nothing herein should be construed as constituting an additional warranty.
+# Open Text shall not be liable for technical or editorial errors or omissions contained herein.
+# The information contained herein is subject to change without notice.
+#
+# END COPYRIGHT NOTICE
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: idol-factbank-postgres-init
+data:
+  {{- $ctx := . }}
+  {{- range tuple "factbank_unified.sql" "01-codings.sql" "02-facts.sql"}}
```

```suggestion
  {{- range tuple "factbank_unified.sql"}}
```
Remove reference to non-present files